[Bug Report] 1.2.0-beta.4  Error - Cannot find module 'element-plus/es/utils/types'

<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

### Element Plus version
1.2.0-beta.4

### OS/Browsers version
Any

### Vue version
3.2.23

### Reproduction Link
https://jsfiddle.net/

### Steps to reproduce
//bash


 yarn add element-plus


//main.ts


 import ElementPlus from 'element-plus'
 import 'element-plus/dist/index.css'
 createApp(App).use(ElementPlus)


//bash


 build

element-plus@1.2.0-beta.4/es/utils/types.d.ts   File does not exist on npm and all cdn.
Only types.mjs and types.mjs.map files exist

### What is Expected?
Done

### What is actually happening?
error TS2307: Cannot find module 'element-plus/es/utils/types' or its corresponding type declarations.

<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

--- a/packages/components/config-provider/index.ts
+++ b/packages/components/config-provider/index.ts
@@ -3,3 +3,5 @@ import { ConfigProvider } from './src'
 
 export const ElConfigProvider = withInstall(ConfigProvider)
 export default ElConfigProvider
+
+export * from './src/config-provider'

--- a/packages/tokens/config-provider.ts
+++ b/packages/tokens/config-provider.ts
@@ -1,7 +1,7 @@
-import type { configProviderProps } from '@element-plus/components/config-provider/config-provider'
+import type { configProviderProps } from '@element-plus/components/config-provider'
 import type { InjectionKey, ExtractPropTypes } from 'vue'
 
-export type ConfigProvdierContext = ExtractPropTypes<typeof configProviderProps>
+export type ConfigProviderContext = ExtractPropTypes<typeof configProviderProps>
 
-export const configProviderContextKey: InjectionKey<ConfigProvdierContext> =
+export const configProviderContextKey: InjectionKey<ConfigProviderContext> =
   Symbol()

--- a/packages/utils/types.ts
+++ b/packages/utils/types.ts
@@ -44,7 +44,7 @@ export type Indexable<T> = {
 
 export type Hash<T> = Indexable<T>
 
-export type TimeoutHandle = ReturnType<typeof global.setTimeout>
+export type TimeoutHandle = number
 
 export type ComponentSize = 'large' | 'medium' | 'small' | 'mini'
 

